src/utils/getAWSRegion.js

```diff
@@ -0,0 +1,6 @@
+const getAWSRegion = () => {
```

Comment:
Does that function even work in staging? I think you should rather use the `networkResources` part of the redux store and add region there?

Reply:
Okay so maybe i can make this happen in the ssr along with the environment